src/lib.rs

```diff
@@ -8,7 +8,7 @@
 // - Deirdre Connolly <deirdre@zfnd.org>
 // - Henry de Valence <hdevalence@hdevalence.ca>
 
-#![doc(html_root_url = "https://docs.rs/reddsa/0.1.0")]
+#![doc(html_root_url = "https://docs.rs/reddsa/0.2.0")]
 #![cfg_attr(feature = "nightly", feature(external_doc))]
```

Comment:
`feature(external_doc)` has been removed, see e.g. https://github.com/clap-rs/clap/issues/2618 for how to fix it.

Reply:
It's been removed in latest releases, but not as of MSRV 1.51, which is what we need for `reddsa` currently. Once the stack is moved to MSRV 1.56 (which will happen soon, but not before `zcashd` 4.7.0) we can re-evaluate this.

Reply:
(The CI failure here is because `redjubjub` doesn't track an MSRV, and this crate is a fork of that one in which I've only made the minimum changes necessary. I haven't delved into ZF's CI stack in particular.)

Reply:
Resolved